test/integration/fatality_notice_test.rb

```diff
@@ -0,0 +1,116 @@
+require 'test_helper'
+
+class FatalityNoticeTest < ActionDispatch::IntegrationTest
+  test "typical fatality notice" do
+    setup_and_visit_content_item('fatality_notice')
+
+    assert page.has_title?(
+      "Sir George Pomeroy Colley killed in Boer War - Fatality notice - GOV.UK"
+    )
+
+    assert page.has_css?(
+      "meta[name='description'][content='It is with great sadness that the Ministry of Defense must confirm that Sir George Pomeroy Colley, died in battle in Zululand on 27 February 1881.']",
```

Comment:
We've spelt "Ministry of Defense" wrong in the content schema example. I'll fix!